### ISSUE TITLE
Cannot individually select SIM for sending SMS

### ISSUE THREAD

**Issue by 0x1a8510f2** [GrapheneOS/Messaging#212]
I'm on a Pixel 8 Pro running GrapheneOS 2024080600 (current latest), with a dual SIM set up consisting of one physical SIM and one eSIM.

When I try to send an SMS message, I get a toast saying "No preferred SIM selected for sending SMS messages".

The normal solution to this, would be to hold the send button until the two SIMs pop up, and select one of them. This works fine, but unfortunately to no effect because when I hit the send button again, I am met with the same toast.

Instead, I have to go into settings, and select a global preferred SIM for sending SMS messages, which is quite inconvenient when frequently switching between the two.

**Comment by buffer51** [comment]
@0x1a8510f2 @corcom does this still happen for you?

I have a similar setup, but I don't have the option to "ask every time" for SMS like there is for phone calls. So one preferred has to be set.

Then inside the Messaging app, I can switch between the 2 SIMs easily.

**Comment by 0x1a8510f2** [comment]
Yeah it looks like I no longer have "ask every time" either. But it seems like I can now switch between the SIMs in the app, so I guess this could be considered resolved?

**Comment by buffer51** [comment]
Yes agreed. I can't do it myself but hoping someone else can.

**Comment by buffer51** [comment]
@thestinger this issue can be closed too, because the "Ask every time" option for SMS no longer exists.
